test/Model.js

```diff
@@ -2137,6 +2137,240 @@ describe('Model', function (){
               });
             });
           });
+
+          it('Update without updateTimestamps (default)', async function () {
+            const cats = [...Array(10)]
+              .map((_, i) => new Cats.Cat4({ id: i + 1, name: 'Tom_' + i }));
+
+            const result = await Cats.Cat4.batchPut(cats);
+            should.exist(result);
+
+            const timestamps = {};
+            cats.forEach((cat, i) => {
+              const { id, myLittleUpdatedAt } = cat;
+              cat.name = 'John_' + i;
+              timestamps[id] = new Date(myLittleUpdatedAt);
+            });
+
+            await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
I just realized one more thing I forgot to mention. Why not make this something much shorter? Like 5 or even 1? Is there a specific reason why we have to wait a full second here? Waiting just a few milliseconds should ensure that the timestamps are later than the original value, correct?

If we decide to edit it we have to make sure we do it throughout these new tests.

Reply:
Yes, I tried making it shorter, and the tests would sometimes fail. Seems like when timestamps get updated milliseconds get stripped away either on the original objects after they saved or on the results of batchGet, not sure which one right now. I didn't have time to investigate this, so I don't know if this is the result of floor, ceiling, or round, so to be safe I made it wait a full second. This made the tests deterministic, no chance of failure anymore. I might be mistaken, but I think the same timeout is used for other tests around timestamps in the test suit too.